pom.xml

```diff
@@ -35,6 +35,12 @@
     <packaging>pom</packaging>
 
     <properties>
+        <!-- Maven -->
+        <maven.version>3.9.7</maven.version>
```

Comment:
Maven wrapper is using 3.9.6; https://github.com/keycloak/keycloak/blob/main/.mvn/wrapper/maven-wrapper.properties#L17

Reply:
Split this out into a separate PR (see #30502). I'll remove this change after that code is merged in.

Reply:
Done, this version is no longer upgraded. I'll resolve this one.